script.js

```diff
@@ -29,14 +29,37 @@ const form = document.getElementById('myForm');
 
 form.addEventListener('submit', (e) => {
   e.preventDefault();
-  const email = document.querySelector('.email-input').value;
-  const suggestedEmail = document.querySelector('.email-input').value.toLowerCase();
+
+  const emailInput = document.querySelector('.email-input');
+  const email = emailInput.value;
+  const suggestedEmail = email.toLowerCase();
+
+  const myStorage = {
+    name: document.getElementById('fname').value,
+    email: document.querySelector('.email-input').value,
+    message: document.getElementById('message').value,
+  };
 
   if (email === suggestedEmail) {
+    localStorage.setItem('myStorage', JSON.stringify(myStorage));
     form.submit();
```

Comment:
You did well with this milestone and your effort is highly commendable, please ensure to follow the project requirement judiciously as your form doesn't save data when the page is reloaded which is a major requirement for this milestone. The data inputted in the form should be saved to the local storage and gotten from there after page is reloaded,please do well to implement this for your project 😄 

project requirement :

![image](https://github.com/katarighe/my-portfolio/assets/20537712/6e5f49dc-11a6-4607-9545-36fa8329f604)